src/main/resources/application.properties

```diff
@@ -14,9 +14,15 @@ spring.datasource.username = ${DATABASE_USER}
 spring.datasource.password = ${DATABASE_PASSWORD}
 spring.jpa.database-platform=org.hibernate.dialect.Oracle12cDialect
 
+spring.flyway.locations = classpath:db/migration
+spring.flyway.enabled = false
+spring.flyway.url = jdbc:postgresql://${LOCAL_DATABASE_HOST}:${LOCAL_DATABASE_PORT}/${LOCAL_SERVICE_NAME}
+spring.flyway.user = ${LOCAL_DATABASE_USER}
+spring.flyway.password = ${LOCAL_DATABASE_PASSWORD}
```

Comment:
Why is not enabled? How about changing the `LOCAL_SERVICE_NAME` variable to something more Postgres like? For example, `LOCAL_DATABASE_NAME`?

Reply:
You don't need to provide anything apart from the `spring.flyway.enabled` as flyway will acquire the parameters from JPA automatically.  I would suggest removing everything (including the enabled parameter) and keeping the default value, letting spring handle it for us.

Reply:
At the moment, we're still working under the assumption that the application will access two databases, and the datasource that's currently configured (the Oracle database) isn't the one to be versioned with Flyway.

Reply:
Updates on this. We have green light to start a new application only for the Postgres database. I imagine that we can close this PR and reopen it on the new repository. @annibalsilva